README.md

```diff
@@ -76,9 +82,6 @@ trusted TLS certificate.
 (optional) Limit for the maximum number of operations performed concurrently.
 Currently this only affects file uploads in the `deploynodeapp` command. Defaults to 4.
 
-`--json  -j`
```

Comment:
Why are we removing this?

Reply:
We're not, my bad, I wasn't sure what it was for, but now I do.
I'll add it back.